JSDemos/Demos/FieldSet/Overview/description.md

```diff
@@ -1 +1,15 @@
-DevExtreme comes with the **&laquo;dx-fieldset&raquo;** CSS class that defines the appearance of&nbsp;an&nbsp;element displaying a&nbsp;list of&nbsp;field-value pairs. Each fieldset item is&nbsp;displayed within a&nbsp;separate field element, which has the **&laquo;dx-field&raquo;** class. The field element may include label and value elements defined by&nbsp;the **&laquo;dx-field-label&raquo;** and **&laquo;dx-field-value&raquo;** classes, respectively.
\ No newline at end of file
+DevExtreme comes with CSS classes that allow you to define the appearance of list-like structures that contain multiple field items - field sets.
+
+## A Simple Field Set
+
+To create a basic field set, use the [dx-fieldset](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset) CSS class for the main container element. Next, use the [dx-fieldset-header](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-fieldset-header) class to specify the field set header. The header element can hold plain text, UI components, or custom markup. 
+
+Create field set items as separate elements with the [dx-field](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field) class. The field element may include label and value elements. Define the label element by the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class. The value element uses the [dx-field-value-static](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-value-static) class and can display plain text or custom markup. 
```

Comment:
> The field element **may** include label and value elements.

The word "may" holds a great degree of uncertainty. Use "can" instead.

I cannot say I understand this sentence:

> Define the label element by the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class.

Reply:
I tried to rewrite the last sentence:

To define the label element, use the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class.

Apply the [dx-field-label](/Documentation/ApiReference/UI_Components/CSS_Classes/#dx-field-label) class to the label element.